### DISCUSSION TITLE
Sylt Shuttle lohnt sich nicht

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#164] [General]
Vorweg: Ich weiß, ich habe den Ertrag selbst so festgelegt 😂

Ich habe gerade mal überlegt, mir einen Zug für den Sylt Shuttle anzuschaffen, der würde mich rund 3M Plops kosten (ist aber auch recht teuer).
Eine realistischere Variante wäre ein Zug mit 2x 218, 2 Dostos + 6 Autowagen (die braucht man mindestens).
Die kostet 2000K Plops, was sich bei der aktuellen Auszahlung von 80K erst nach vielen Fahrten lohnen würde.

Also wäre eine Erhöhung der Auszahlung auf so 200K sinnvoll, was aber widerum für 40km doch sehr viel wäre.
Wäre dann ein Kompromiss von so 140K-160K sinnvoll?

**Comment by marhei** [comment]
Klingt fair, ja…
